Source/Core/VideoBackends/OGL/main.cpp

```diff
@@ -176,6 +177,10 @@ bool VideoBackend::FillBackendInfo()
     return false;
   }
 
+  // S3TC for native DXT1/3/5 custom texture support.
+  g_Config.backend_info.bSupportsST3CTextures =
+      GLExtensions::Supports("GL_EXT_texture_compression_s3tc");
```

Comment:
All other functions of this kind are in Render.cpp. I don't like them there, but we should not start to split them